src/api_fns.py

```diff
@@ -40,23 +41,34 @@ def get_env_info(env:str=ENV, config_file:str='config/config.yaml')->str:
     env_port = config_dict[env]['API port']
     return env_host, env_port
 
-
-def generate_paths(blob_hash:str) -> list:
-    bucket = ['blobs/']
-    return [b+blob_hash for b in bucket]
+def duplicate(blob_hash: str, cur) -> bool:
+    """ 
+    Checks if blob is already in Cabinet
+    """
+    cur.execute("SELECT COUNT(1) FROM blob WHERE blob_hash = %s", (blob_hash,))
```

Comment:
this doesnt work if you want to save your blob to more than one destination.

Reply:
added ability to save to new locations